airbyte-ci/connectors/live-tests/README.md

```diff
@@ -119,7 +119,8 @@ You can run the existing test suites with the following command:
 #### With local connection objects (`config.json`, `catalog.json`, `state.json`)
 ```bash
 poetry run pytest src/live_tests/regression_tests \ 
---connector-image=airbyte/source-faker \
+ --connector-image=airbyte/source-faker \
+ --connection-id=<CONNECTION-ID> \
```

Comment:
Can we provide a way to easily get a working connection-id for this command? I think this would be great because the users going through the readme.md will probably be complete beginners

Reply:
We'll have some tooling for automatically picking a connection ID merged soon. But I think that people should really still be using the onboarding doc. Explanation [here](https://airbytehq-team.slack.com/archives/C06GNN6BERK/p1713906384719329?thread_ts=1713900397.366909&cid=C06GNN6BERK).